.github/workflows/test.yml

```diff
@@ -1,26 +0,0 @@
-name: Test
-on: pull_request
-
-jobs:
-  build:
-    name: Test with coverage
```

Comment:
@DiederikvandenB can you please remove the required status check/branch protection rule? I think it is also what prevents me from pushing to `master` - see https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule